### PR TITLE
:book: Update details on CII->OpenSSF Best Practices badge

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Update `docs/checks/internal/checks.yaml` and the generated file `docs/checks.md` to note that "CII Best Practices badge" is now the OpenSSF Best Practices badge.

Signed-off-by: David A. Wheeler <dwheeler@dwheeler.com>

#### What kind of change does this PR introduce?

(Is it a bug fix, feature, docs update, something else?)

- [ ] PR title follows the guidelines defined in our [pull request documentation](https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process)

#### What is the new behavior (if this is a feature change)?**

- [ ] Tests for the changes have been added (for bug fixes/features)

#### Does this PR introduce a user-facing change?

For user-facing changes, please add a concise, human-readable release note to
the `release-note`

(In particular, describe what changes users might need to make in their
application as a result of this pull request.)

```release-note

```

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]

File: docs/checks.md [hunk 1]
```diff
@@ -112,19 +112,19 @@ This check tries to determine if the project is published as a package that othe
...
 *Limitations:* This check should only apply to applications, as libraries shouldn't normally have enforced pinned dependencies. Unfortunately, Scorecard currently can't detect if a project is a library or application. Even when [application detection is added](https://github.com/ossf/scorecard/issues/689), it's always possible for an automated tool like Scorecard to incorrectly categorize software (especially in projects that include both libraries and applications).
-You can learn more about dependencies for projects on GitHub using [GitHub dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph). 
+If the project is hosted on GitHub, You can learn more about dependencies for projects using [GitHub dependency graph](https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph).
```
Comment (olivekl) [inline docs/checks.md:127]: Tiny nit, lowercase on "Y" of "You"

**Review by olivekl — APPROVED** [review]
LGTM, thanks!

**Comment by naveensrinivasan** [comment]
@david-a-wheeler  The  PR has conflicts.

File: docs/checks.md (hunk 1 above)
Comment (david-a-wheeler) [reply docs/checks.md:127]: You're right! In the source it's split across lines so I didn't notice it. I'll post a fix.

**Comment by david-a-wheeler** [comment]
Drat, I used the wrong tip. Sorry about that. I'm closing this. Please use this instead:

https://github.com/ossf/scorecard/pull/2239

**Pull request closed without merging**